src/app/clusters/group-key-mgmt-server/group-key-mgmt-server.cpp

```diff
@@ -438,7 +438,7 @@ bool emberAfGroupKeyManagementClusterKeySetReadCallback(
     }
     response.groupKeySet.epochKey2.SetNull();
 
-    CHIP_ERROR err = commandObj->AddResponseData(commandPath, response);
+    CHIP_ERROR err = commandObj->AddResponse(commandPath, response);
     if (CHIP_NO_ERROR != err)
```

Comment:
This is going to add a status again on failure, which again does not make sense.  Please remove the emberAfSendImmediateDefaultResponse call.